robot/rospackages/src/image_scp/image_scp.py

```diff
@@ -0,0 +1,79 @@
+import fnmatch
+import os
+from paramiko import SSHClient
+from scp import SCPClient, SCPException
+import argparse
+from config import (host, user, remote_path_1)
+"""
+Script takes in a directory name as a command line argument. This directory should contain
```

Comment:
Unforuntately I think this approach isn't the most efficient, it's probably best to make the script target a directory that is remote, the config file can stay local though.